src/actinia_core/core/geodata_download_importer.py

```diff
@@ -215,22 +215,35 @@ def get_file_rename_command(file_path, file_name):
         return p
 
     @staticmethod
-    def get_raster_import_command(file_path, raster_name):
+    def get_raster_import_command(file_path, raster_name, resample=None,
+                                  resolution=None, resolution_value=None):
         """Generate raster import process list that makes use of r.import
 
         Args:
             file_path:
             raster_name:
+            resample:
```

Comment:
Maybe it wouldn't hurt here to add the datatype and maybe a descriptive sentence like https://github.com/mundialis/actinia_core/blob/main/src/actinia_core/core/common/api_logger.py#L134-L136

Reply:
I added some description here , however not sure about the datatype for `resolution_value`. The value is given as string but needs to be convertable to float to use in r.import.

Reply:
For the method is has to be a string. Otherwise you has to change `executable_params.append("resolution_value=%s" % resolution_value)` to ` executable_params.append("resolution_value=%d" % resolution_value)`.

Reply:
Thanks, I set it to `str` then.